packages/ui-components/src/types.ts

```diff
@@ -22,14 +22,6 @@ export * from './components/tree-item/tree-item.types';
 export * from './components/tree/tree.types';
 export * from './components/badge/badge.types';
 export * from './components/modal/modal.types';
-export * from './components/calendar/calendar.types';
```

Comment:
hmmm, I have doubts about removing some types of here that can be used inside of the application. Please review the ones that aren't used.

Reply:
Yeah, probably I lost the export with the renaming and forgot to check here! Will take a look 🙏